data/net/minecraft/util/KeyDispatchDataCodec.mapping

```diff
@@ -1,5 +1,16 @@
 CLASS net/minecraft/util/KeyDispatchDataCodec
+	COMMENT A codec for serializing and deserializing key dispatch data in Minecraft.
+	COMMENT <p>
+	COMMENT @param <A> the type of data that this codec handles.
```

Comment:
```suggestion
	COMMENT A wrapper around a {@code Codec<T>}, who's purpose is to indicate that the codec is not to be used directly, and instead used by another codec using {@code .dispatch()}.
```

I don't think the doc, as written, really says anything about this class. That said, I took a long hard look and I don't understand what on earth this was actually meant to be used for, so this is the best explanation that I could come up with.